docs/pages/responsive-accordion-tabs.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Responsive Accordion Tabs
+description: Use the Markup of a Accordion or Tabs and create a Responsive Accordion Tabs.
+---
+
+## Basics
+
+Eighter <a href="accordion.html">Accordion Markup</a> or <a href="tabs.html">Tabs Markup</a> can be used for responsive switching these components
```

Comment:
"Either the Accordion markup or the Tabs markup can be used to responsively switch between the two components at different breakpoints."